type checking javascript files 줄맞춤, 형식 수정. (#194)
yeonjuan/Typescript-Handbook-ko: type-checking-javascript-files.md

`.ts` 파일에서, 변수 선언을 초기화하는 객체 리터럴은 선언에 해당 타입을 부여합니다.
원본 리터럴에 명시되어 있지 않은 새 멤버는 추가될 수 없습니다.

> 이 규칙은 `.js` 파일에선 완화됩니다; 객체 리터럴은 원본에 정의되지 않은 새로운 프로퍼티를 조회하고 추가하는 것이 허용되는 확장 가능한 타입(인덱스 시그니처을 갖습니다.

Reviewer: ```suggestion
이 규칙은 `.js` 파일에선 완화됩니다; 객체 리터럴은 원본에 정의되지 않은 새로운 프로퍼티를 조회하고 추가하는 것이 허용되는 확장 가능한 타입(인덱스 시그니처)을 갖습니다.
```

> ## `arguments`의 사용으로부터 추론된 var-args 매개변수의 선언 (Var-args parameter declaration inferred from use of `arguments`)

Reviewer 2: ```suggestion
## `arguments` 사용으로부터 추론된 var-args 매개변수 선언 (Var-args parameter declaration inferred from use of `arguments`)
```

> `arguments`의 참조를 참조하는 본문을 가진 함수는, 암묵적으로 var-args 매개변수(예: `(...arg: any[]) => any`)를 갖는 것으로 간주합니다. JSDoc의 var-args 구문을 사용하여 인수의 타입을 지정할 수 있습니다.

Reviewer 2: ```suggestion
`arguments` 참조를 참조하는 본문을 가진 함수는, 암묵적으로 var-args 매개변수(예: `(...arg: any[]) => any`)를 갖는 것으로 간주합니다. JSDoc의 var-args 구문을 사용하여 인수의 타입을 지정할 수 있습니다.
```

### 함수 호출에서 (In function calls)

> 제네릭 함수의 호출은 인수를 사용해 타입 매개변수를 추론합니다. 때때로 이 과정은 추론 소스가 부족하여 어떠한 타입도 추론하지 못하는 경우가 있습니다; 이러한 경우, 매개변수 타입은 기본적으로 `any`입니다. 예를 들어:

Reviewer 2: ```suggestion
제네릭 함수의 호출은 인수를 사용해 타입 매개변수를 추론합니다. 때때로 이 과정은 추론 소스가 부족하여 어떠한 타입도 추론하지 못하는 경우가 있습니다; 이러한 경우, 매개변수 타입은 기본적으로 `any`입니다. 예를 들어:
```

var arrayLike;
```

앞의 두 타입은 TypeScript의 `{ [x: string]: number }` 그리고 `{ [x: number]: any }`와 동일합니다. 컴파일러는 두 구문을 모두 이해할 것입니다.

> 함수 타입은 TypeScript 또는 Closure 구문을 사용하여 지정할 수 있습니다:

Reviewer: ```suggestion
함수 타입은 TypeScript 또는 클로저 구문을 사용하여 지정할 수 있습니다:
```
[제안]
클로저는 많이 쓰는 표현이라 굳이 원어로 안적어도 될 것 같은데 어떤가요?
